packages/expo/src/__tests__/Notifications-test.android.ts

```diff
@@ -0,0 +1,96 @@
+import { NativeModules } from 'react-native';
+
+import Notifications from '../Notifications/Notifications';
+
+jest.mock('react-native/Libraries/EventEmitter/RCTDeviceEventEmitter', () => {
+  const { EventEmitter } = require('fbemitter');
+  return new EventEmitter();
+});
+jest.useFakeTimers();
+
+const mockedScheduledNotifIOS = {
+  title: 'Mock notification',
+  body: 'hello',
+};
+it(`properly passes time as millis when scheduling notification on Android`, async () => {
+  const spy = jest.fn();
```

Comment:
I know you didn't write this but I'd prefer we used the Jest mocks already provided. `NativeModules.ExponentNotifications.scheduleLocalNotification` should already be a mock function, and between tests we can call `jest.clearAllMocks`.